### ISSUE TITLE
fix: WS?

### ISSUE THREAD

**Issue by YohannesTz** [RobiMez/sma#12]
Web-socket much much better than polling?

**Comment by RobiMez** [comment]
Yeah i plan to add it but for now i like the simplicity of polling , will add when demand rises. or when polling becomes a bottle neck.
